aten/src/ATen/cpu/vec256/vec256_float.h

```diff
@@ -312,7 +312,16 @@ Vec256<float> inline operator/(const Vec256<float>& a, const Vec256<float>& b) {
 
 // frac. Implement this here so we can use subtraction
 Vec256<float> Vec256<float>::frac() const {
-  return *this - this->trunc();
+  const auto pos_inf_vec = _mm256_set1_ps(INFINITY);
```

Comment:
Have you tried implementing the vectorized part in the same way you did the serial part using std::isinf? Is the benchmark code using the vectorized or serial part? Do you know if the perf regression exists in both paths?

Reply:
Yes the vectorized part is similar to serial part (since there is no intrinsic similar to `std::isinf`), we manually verify if the check for `INFINITY` and `-INFINITY`.

The benchmark is mostly for the vectorized path as all shapes are fairly large.